src/main/java/org/xstefank/verification/InvalidConfigurationException.java

```diff
@@ -2,7 +2,9 @@
 
 public class InvalidConfigurationException extends Exception {
 
-    public InvalidConfigurationException(String message) {
+    private static final long serialVersionUID = 1L;
```

Comment:
https://stackoverflow.com/questions/888335/why-generate-long-serialversionuid-instead-of-a-simple-1l
http://frequal.com/java/PracticalSerialVersionIdGuidelines.html